views/templates/admin/configuration.tpl

```diff
@@ -68,8 +69,7 @@
     <div class="mj_landing">
         <div align="center">
             <br />
-            <a href="https://www.mailjet.com/support"
-               target="_blank">{l s='If you have a question or if you have a problem, click here to contact support.' mod='mailjet'}</a>
+            {{l s='If you have a question or if you have a problem, click here to contact support.' mod='mailjet'}|html_entity_decode}
```

Comment:
Oh, so you wrote HTML code in a translated text. That's tricky.

You can guess there is a reason if the outputed text is escaped. :wink: